src_assets/macos/assets/apps.json

```diff
@@ -2,5 +2,18 @@
   "env": {
     "PATH": "$(PATH):$(HOME)/.local/bin"
   },
-  "apps": []
+  "apps": [
+    {
+      "name": "Desktop",
+      "image-path": "desktop.png"
+    },
+    {
+      "name": "Steam BigPicture",
+      "output": "steam.txt",
+      "detached": [
+        "open steam://open/bigpicture"
```

Comment:
Does this have the right `$PATH` to actually launch Steam?

Reply:
@brad-richardson confirmed this command worked with the default path.